### PR TITLE
Frontend CLI WIP

### LINKED ISSUE #2: Create CLI client
## Current Situation

We dont have a CLI client yet. we need it. You can use the [click](https://click.palletsprojects.com/en/7.x/) library for this. 

Have a look at the README 
1. https://github.com/ILUGD/Birdman#working
2. https://github.com/ILUGD/Birdman#extras

### PR DESCRIPTION
**Pull request by progmatic-99**
- [ ] Included tests for API changes.
- [x] Included a description of change below.
- [ ] Tried to [squash the commits][squash].
- [x] Created an ISSUE for this PR 
- [x] Add WIP in the title if PR is Work In Progress
### Changes done in this Pull Request
Just created a basic frontend CLI.

- Fixes #2 

### The problem I want to solve or the facility I want to improve is/are

Removed the recursive calls to main()
### My steps to solve it

Changed main.py and added file support so that details are written in the meetup.txt file.

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

File: frontend/main.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    if venue_decided:
+
+        meetup_link = click.prompt("Meetup Link", type=click.STRING)
+        poster_link = click.prompt("Poster Link", type=click.STRING)
+        tg_usrname = click.prompt("Telegram Username", type=click.STRING)
+        talk_decided = click.confirm("Talks Decided")
+
+        if talk_decided:
+            click.echo("Processing...(saving to .birdman/<date>/meetup.txt)\n")
+            click.echo("Join us for the next ILUG-D meetup at <Date/Time> at <Venue>\n")
```
Comment (pythongiant) [inline frontend/main.py]: dont print text. actually save the template to .birdman/<date>/meetup.txt first and then print the file

File: frontend/main.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            click.echo(f"Event Page: {meetup_link}\n")
+
+            choice = click.confirm("Would you like to edit this?")
+
+            if choice:
+                details()
```
Comment (pythongiant) [inline frontend/main.py]: This shouldnt redo the whole process. Instead, open the file(.birdman/<date>/meetup.txt) in nano

**Review by pythongiant — CHANGES_REQUESTED** [review]

File: frontend/main.py (hunk 1 above)
Comment (progmatic-99) [reply frontend/main.py]: what template are you talking about here?

Comment (pythongiant) [reply frontend/main.py]: Join us for the next ILUG-D meetup at <Date>.....  
Use fstrings

**Comment by pythongiant** [comment]
Could you push this to a new branch ?

**Pull request closed without merging**